.github/workflows/vercel-deploy.yml

```diff
@@ -0,0 +1,31 @@
+name: Vercel Deploy
+
+env:
+  VERCEL_ORG_ID: ${{ secrets.VERCEL_ORG_ID }}
+  VERCEL_PROJECT_ID: ${{ secrets.VERCEL_PROJECT_ID }}
+
+on:
+  schedule:
```

Comment:
Is that necessary to re-deploy everyday?
I think `push` and `workflow_dispatch` are enough.